diffsims/tests/crystallography/test_reciprocal_lattice_vector.py

```diff
@@ -108,6 +108,14 @@ def test_repr(self, ferrite_phase):
             "[[ 1.  1.  0.]",
         ]
 
+    @pytest.mark.parametrize("degrees", [True, False])
```

Comment:
`to_polar()` is inherited from `Vector3d` in orix. I see no need to test it here, so I suggest to remove it.

Reply:
Done!